my-app/src/app/shared/components/autocomplete-select/autocomplete-select.component.scss

```diff
@@ -0,0 +1,141 @@
+:host {
+    display: block;
+    box-sizing: border-box;
+    position: relative;
+
+    .text-input-container {
+
+        .text-search-field {
+            display: flex;
+            align-items: center;
+            border: 1px solid gray;
+            width: 100%;
+            background-color: white;
+            border-radius: 2px;
+        }
+
+        input {
```

Comment:
avoid such deep nesting. It will be hard to read and change the styles in the future